src/native/src/3rdparty/glad/include/KHR/khrplatform.h

```diff
@@ -26,7 +26,7 @@
 
```

Comment:
Just fwi this commit also has a regenerated copy of glad with OpenGL 4.5 Core Profile + Extensions compatible with the core profile.